CRM/Core/BAO/CustomValueTable.php

```diff
@@ -714,7 +715,7 @@ public static function setValues(&$params) {
    * @throws Exception
    * @return array
    */
-  public static function &getValues(&$params) {
+  public static function getValues($params) {
     if (empty($params)) {
       return NULL;
```

Comment:
This makes the pass-by-reference illegal.

Reply:
we really haven't hit many issues removing those preceding `&` signs. I did look them up once & they were introduced to 'add php4 compatibility'